packages/blocks/src/lib/navigation-map/components/vector-input.svelte

```diff
@@ -0,0 +1,54 @@
+<script lang='ts'>
+
+import { Label, SliderInput } from '@viamrobotics/prime-core';
+import { createEventDispatcher } from 'svelte';
+
+export let readonly = false;
+export let placeholders = ['0', '0', '0'];
```

Comment:
Also feels like it should be an object keyed by label

Reply:
agreed, fixed!